modyn/supervisor/internal/triggers/trigger.py

```diff
@@ -1,20 +1,37 @@
+import pathlib
 from abc import ABC, abstractmethod
+from typing import Generator
 
 
 class Trigger(ABC):
     def __init__(self, trigger_config: dict) -> None:
         assert trigger_config is not None, "trigger_config cannot be None."
 
     @abstractmethod
-    def inform(self, new_data: list[tuple[int, int, int]]) -> list[int]:
-        """The supervisor informs the trigger about new data.
-        In case the concrete trigger implementation decides to trigger, we return a list of _indices into new_data_.
+    def init_trigger(self, pipeline_id: int, pipeline_config: dict, modyn_config: dict, base_dir: pathlib.Path) -> None:
```

Comment:
I think we can make this a non-abstract method with just a `pass` implementation, since only the drift trigger uses this. Then we don't need the empty overrides in the other triggers.

Reply:
done

Reply:
oops, missed it. Now it's fixed.